parcel_test.go

```diff
@@ -21,67 +22,101 @@ var (
-
-	// get
-	// получите только что добавленную посылку, убедитесь в отсутствии ошибки
-	// проверьте, что значения всех полей в полученном объекте совпадают со значениями полей в переменной parcel
-
-	// delete
-	// удалите добавленную посылку, убедитесь в отсутствии ошибки
-	// проверьте, что посылку больше нельзя получить из БД
+	// add (отсутствие ошибки, наличие идентификатора)
+	number, err := store.Add(parcel)
+	require.NoError(t, err, "failed to add parcel")
+	assert.NotEmpty(t, number, "parcel number should not be empty")
+
+	// get (отсутствие ошибки, значения полей в объекте = значения полей в parcel)
+	get, err := store.Get(number)
+	require.NoError(t, err, "failed to get parcel")
+	assert.Equal(t, parcel.Client, get.Client)
+	assert.Equal(t, parcel.Address, get.Address)
+	assert.Equal(t, parcel.Status, get.Status)
+	assert.Equal(t, parcel.Created_At, get.Created_At)
+
+	// delete (отсутствие ошибки, посылку больше нельзя получить из БД)
+	err = store.Delete(number)
+	require.NoError(t, err, "failed to delete parcel")
+	_, err = store.Get(number)
+	assert.Error(t, err, "error when getting deleted parcel")
 }
 
 // TestSetAddress проверяет обновление адреса
 func TestSetAddress(t *testing.T) {
 	// prepare
-	db, err := // настройте подключение к БД
+	db, err := sql.Open("sqlite", "tracker.db")
+	require.NoError(t, err, "failed to open database")
+	defer db.Close()
+
+	store := NewParcelStore(db)
+	parcel := getTestParcel()
 
-	// add
-	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
+	// add (отсутствие ошибки, наличие идентификатора)
+	number, err := store.Add(parcel)
+	require.NoError(t, err, "failed to add parcel")
+	assert.NotEmpty(t, number, "parcel number should not be empty")
 
-	// set address
-	// обновите адрес, убедитесь в отсутствии ошибки
+	// set address (отсутствие ошибки)
 	newAddress := "new test address"
+	err = store.SetAddress(number, newAddress)
+	require.NoError(t, err, "failed to set new address")
 
 	// check
-	// получите добавленную посылку и убедитесь, что адрес обновился
+	get, _ := store.Get(number)
+	assert.Equal(t, newAddress, get.Address)
 }
 
 // TestSetStatus проверяет обновление статуса
 func TestSetStatus(t *testing.T) {
 	// prepare
-	db, err := // настройте подключение к БД
+	db, err := sql.Open("sqlite", "tracker.db")
+	require.NoError(t, err, "failed to open database")
+	defer db.Close()
 
-	// add
-	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
+	store := NewParcelStore(db)
+	parcel := getTestParcel()
+
+	// add (отсутствие ошибки, наличие идентификатора)
+	number, err := store.Add(parcel)
+	require.NoError(t, err, "failed to add parcel")
+	assert.NotEmpty(t, number, "parcel number should not be empty")
 
-	// set status
-	// обновите статус, убедитесь в отсутствии ошибки
+	// set status (отсутствие ошибки)
+	newStatus := "Delivered"
```

Comment:
Если используем общий статус, то лучше сделать константу
```suggestion
	newStatus := ParcelStatusDelivered
```

Reply:
Спасибо за подробное ревью! Код исправила и отправила повторно.